flink-core/src/main/java/org/apache/flink/configuration/HighAvailabilityOptions.java

```diff
@@ -167,6 +167,8 @@ public class HighAvailabilityOptions {
                     .withDescription(
                             "Defines the number of connection retries before the client gives up.");
 
+    /** @deprecated Don't use this option anymore. It has no effect on Flink. */
+    @Deprecated
     @Documentation.Section(Documentation.Sections.EXPERT_ZOOKEEPER_HIGH_AVAILABILITY)
     public static final ConfigOption<String> ZOOKEEPER_RUNNING_JOB_REGISTRY_PATH =
```

Comment:
Maybe, add a description stating that the config parameter is deprecated and not used anymore.

Reply:
The description on deprecation has already been added in the comment, do you mean to add description using the . withDescription as well?

Reply:
Yes, so that users who check the website would be also aware of the change.

Reply:
But the doc generator won't generate docs for deprecated options, so users won't see this on the website.

Reply:
I think we'd better generate docs for deprecated options as well to offer proper upgrading guidance in the doc, otherwise users have to either carefully read the release notes or to directly look up the source code. I've created a JIRA to track this: https://issues.apache.org/jira/browse/FLINK-33240. Maybe we can work on that issue first, and come back to this one.

Reply:
hm, I wanted to reproduce it but failed. How did you generate the docs?
I tried the what's documented in [docs/README.md:62](https://github.com/apache/flink/blob/5c1e9f3b1449cb77276d578b344d9a69c7cf9a3c/docs/README.md#L62):
```bash
mvn -Pgenerate-config-docs install -Dfast -DskipTests
```
and what's documented in [flink-docs/README.md:44](https://github.com/apache/flink/blob/7bebd2d9fac517c28afc24c0c034d77cfe2b43a6/flink-docs/README.md#L44)
```bash
mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests
```
Both approaches didn't regenerated the HTML files. What am I missing? :thinking:

Reply:
I was wrong. I didn't properly revert the files before running the command again.

Reply:
I've added description in a new commit